README.md

```diff
@@ -90,9 +90,9 @@ logging_flows:
   available keys
   - `kernel_message`: Load `imklog` if set to `true`. Default to `false`.
   - `use_imuxsock`: Use `imuxsock` instead of `imjournal`. Default to `false`.
-  - `journal_ratelimit_burst`: The value is set to imjournal RateLimit.Burst. Default to 20000.
-  - `journal_ratelimit_interval`: The value is set to imjournal RateLimit.Interval. Default to 600.
-  - `journal_persist_state_interval`: The value is set to imjournal PersistStateInterval. Default to 10.
+  - `ratelimit_burst`: The value is set to imjournal RateLimit.Burst. Default to 20000 if use_imuxsock is false. Default to 200 if use_imuxsock is true.
+  - `ratelimit_interval`: The value is set to imjournal RateLimit.Interval. Default to 600 if use_imuxsock is false. Default to 0 if use_imuxsock is true.
```

Comment:
The unit is seconds?

Reply:
Yes, and 0 means not setting. I'm adding the info.